src/SFA.DAS.Reservations.Web/Views/Reservations/Confirmation.cshtml

```diff
@@ -43,10 +43,15 @@
             Apprenticeship training
         </dt>
         <dd class="govuk-summary-list__value">
-            @if (Model.Course != null)
+            
+            @if (Model.Course != null && !Model.Course.Title.ToUpper().Equals("UNKNOWN"))
             {
                 @: @Model.Course.Title - Level: @Model.Course.Level
             }
+            else
+            {
+                @: Unknown
```

Comment:
Can this not be Model.Course.Title? I would have thought the additional if clause isn't reachable. It would be preferable to move this logic out and to make the CourseDescription a property on the model.